contracts/L2/token/LivepeerToken.sol

```diff
@@ -0,0 +1,45 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
+import {ERC20, ERC20Permit} from "@openzeppelin/contracts/token/ERC20/extensions/draft-ERC20Permit.sol";
+import {ILivepeerToken} from "./ILivepeerToken.sol";
+
+contract LivepeerToken is ILivepeerToken, AccessControl, ERC20Permit {
+    bytes32 public constant MINT_CONTROLLER = keccak256("MINT_CONTROLLER");
```

Comment:
Nit suggestion to follow the role naming convention already used in AccessControl:

```suggestion
    bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
```